src/modules/format.F90

```diff
@@ -0,0 +1,27 @@
+module format
+!
+!       This module contains common format specifiers for printing output.
+!
+       use constants, only : maxStringLen
+       implicit none
+       save
+
+       ! An arbitrary number of strings
```

Comment:
I think the word "arbitrary" could be applied to the other ones also, or we should clearly explain what the `*` is, just to make it more clear as this notation is newer and less used. Something like the following could be added. Feel free to adjust this as needed.
"The `*` in a pattern means that it is applied as often as there are input arguments matching this pattern. 
For example, the `(*(A, 1X))` means that this formatting pattern is applied N times if you have N string that you want to be separated by as space. 
Similarly, the `(*(A, ES12.5))` expects N string, float pairs as an input."